frontend/src/metabase/writeback/utils.ts

```diff
@@ -19,6 +21,27 @@ export const isDatabaseWritebackEnabled = (database?: IDatabase | null) =>
 export const isWritebackSupported = (database?: Database | null) =>
   !!database?.hasFeature(DB_WRITEBACK_FEATURE);
 
+const AUTOMATIC_DATE_TIME_FIELDS = [
+  TYPE.CreationDate,
+  TYPE.CreationTemporal,
+  TYPE.CreationTime,
+  TYPE.CreationTimestamp,
+
+  TYPE.DeletionDate,
+  TYPE.DeletionTemporal,
+  TYPE.DeletionTime,
+  TYPE.DeletionTimestamp,
+
+  TYPE.UpdatedDate,
+  TYPE.UpdatedTemporal,
+  TYPE.UpdatedTime,
+  TYPE.UpdatedTimestamp,
+];
```

Comment:
@kdoh, I noticed we also have semantic types like `type/JoinDate` and `type/JoinTimestamp.` I suppose they're essentially "Created At" but for user account records. Should we also include them here, so they're not shown in create/update forms?

Reply:
Yeah I think that'd be smart, but doesn't need to block merging this in my opinion.